src/logging/wal_logger.cpp

```diff
@@ -0,0 +1,101 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// wal_logger.cpp
+//
+// Identification: src/logging/wal_logger.cpp
+//
+// Copyright (c) 2015-18, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
```

Comment:
Probably need tests to make sure that logger thread is actually writing logs to the disk before sending the commit and abort message back to the transaction.